src/xdocs/config_metrics.xml

```diff
@@ -101,6 +101,23 @@
         <source>
 &lt;module name=&quot;BooleanExpressionComplexity&quot;/&gt;
         </source>
+        <p>Code Example:</p>
+        <source>
+public class Test
+{
+  public static void main(String ... args)
+  {
+    boolean a = true;
+    boolean b = false;
+
+    boolean c = (a &amp; b) | (b &amp; a);
+    //OK, (&amp;,|) are counted as 1 which gives the total complexity of 3(max allowed 3)
+
+    boolean d = (!a &amp;&amp; b) ^ (a || !b) &amp;&amp; a;
+    //violation, (&amp;&amp;,^,||) are counted as 1 which gives total of 4(max allowed 3)
```

Comment:
The comment `// violation` should be on the same line as it would be reported in checkstyle. If more room is needed to explain reasoning, then that can be on the next line in its own comment.

Reply:
@rnveach can i use multiline comments like /* */
Also if // violation and // ok should be on same line then the next line comment should begin from start of the line or below it

Reply:
Done